jerry-core/ecma/builtin-objects/ecma-builtin-helpers-json.c

```diff
@@ -158,58 +147,43 @@ ecma_builtin_helper_json_create_separated_properties (ecma_collection_header_t *
  *         Returned value must be freed with ecma_free_value.
  */
 ecma_value_t
-ecma_builtin_helper_json_create_formatted_json (ecma_string_t *left_bracket_p, /**< left bracket*/
-                                                ecma_string_t *right_bracket_p, /**< right bracket*/
+ecma_builtin_helper_json_create_formatted_json (ecma_char_t left_bracket, /**< left bracket character */
```

Comment:
Why don't we use `lit_utf8_byte_t` for `left_bracket` and `right_bracket`? It looks like you are casting it anyway.

Reply:
The idea was that we don't reduce anything since byte/short/int consumes 1 machine register, and I added an assert. But if you prefer I can change it.

Reply:
IMHO the code will be a bit more readable that way. Please change it.